### PR TITLE
feat: add cmlm encoder

### PR DESCRIPTION
**Pull request by carlosb1**
It adds universal cmlm encoder

### CODE REVIEW

**Comment by carlosb1** [comment]
@JoanFM , I am not sure, but it doesn't seem related with my rebase. I did another one, (it seems updated with `git log`) and it keeps throwing the same error.

**Review by JoanFM — CHANGES_REQUESTED** [review]
Hey @carlosb1 ,

it looks really good, I think it is the idea to have a potential preprocessor

I do however would like to see a test where a preprocessor is applied.

Also, please bump the manifest version so that the CI and the tests are fired

File: encoders/nlp/UniversalSentenceEncoder/tests/test_universalsentenceencoder.py [hunk 1]
```diff
@@ -75,3 +75,23 @@ def test_save_and_load_config(mocker):
...
+@mock.patch('tensorflow_hub.load', return_value=MockModule())
+def test_get_universal_sentence_encoder(mocker):
+    MODEL_ENCODER = 'https://tfhub.dev/google/universal-sentence-encoder/4'
+    metas = get_metas()
+    encoder = UniversalSentenceEncoder(metas=metas)
+    assert encoder.model_url == MODEL_ENCODER
+
+
+@mock.patch('tensorflow_hub.load', return_value=MockModule())
+def test_get_universal_sentence_encoder_cmlm(mocker):
```
Comment (carlosb1) [inline encoders/nlp/UniversalSentenceEncoder/tests/test_universalsentenceencoder.py:89]: @JoanFM  Here we have a test to validate the preprocess parameter. do we need another one?

**Comment by JoanFM** [comment]
Hey @carlosb1 , it seems a problem of a last commit that I squashed, I guess you trying to rebase is firing the problem again? Can you maybe create a fresh directly from master and copying these changes?

Sorry for the inconvinience

**Review by carlosb1 — COMMENTED** [review]
modified

File: encoders/nlp/UniversalSentenceEncoder/tests/test_universalsentenceencoder.py (hunk 1 above)
Comment (JoanFM) [reply encoders/nlp/UniversalSentenceEncoder/tests/test_universalsentenceencoder.py:89]: I mean that it actually properly encodes some input, here we are only testing initialization and post_init.

**Comment by carlosb1** [comment]
> Hey @carlosb1 ,
> 
> it looks really good, I think it is the idea to have a potential preprocessor
> 
> I do however would like to see a test where a preprocessor is applied.
> 
> Also, please bump the manifest version so that the CI and the tests are fired

The error should be there.

**Pull request closed without merging**